recipes/wildmidi/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conan import ConanFile
+from conan.tools.microsoft import is_msvc
+from conan.tools.files import export_conandata_patches, get, copy, rmdir
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class WildmidiConan(ConanFile):
+    name = "wildmidi"
+    version = "0.4.5"
+    description = "WildMIDI is a simple software midi player which has a core softsynth library that can be used in other applications."
+    license = "LGPL-3.0-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.mindwerks.net/projects/wildmidi"
+    topics = ("midi", "audio", "sound", "multimedia")
```

Comment:
I thing it would help others find this library by including the tags `softsynth`, `synth`, and `music`.

```suggestion
    topics = ("audio", "midi", "multimedia", "music", "softsynth", "sound", "synth")
```